bridge/bridgeapp/views.py

```diff
@@ -123,3 +124,28 @@ def post(self, request, thread_id, resp_id):
                 Response.objects.filter(id=resp_id).delete()
         # redirect to single thread page
         return redirect('thread', thread_id, 0)
+
+
+def register(request):
```

Comment:
Here I suggest we create a `BridgeAuthenticationView` class that handles both registration and login on the same page for:
1. efficiency
2. consistency
We have 3 class models, 3 class views and 2 class forms and not a single functional programming in our code base. A class-based view for both registration and login functionalities does not require extra effort to code up and would serve both of the aforesaid purposes to make the app cleaner, neater more efficient performance-wise.